nix/packages/stub.nix

```diff
@@ -0,0 +1,44 @@
+{ rustPlatform, clippy, fetchFromGitHub, stdenv, lib, runCommand, enableLint ? false }:
+
+rustPlatform.buildRustPackage
+  {
+    pname = "lanzastub";
+    version = "0.3.0";
+    src = runCommand "src" { } ''
+      install -D ${../../rust/stub/Cargo.toml} $out/Cargo.toml
+      install -D ${../../rust/stub/Cargo.lock} $out/Cargo.lock
+      cp -r ${../../rust/stub/src} $out/src
+    '';
+
+    nativeBuildInputs = lib.optional enableLint clippy;
+
+    cargoLock = {
+      lockFile = ../../rust/stub/Cargo.lock;
+    };
+
+    # Necessary because our `cc-wrapper` doesn't understand MSVC link options.
+    RUSTFLAGS = "-Clinker=${stdenv.cc.bintools}/bin/${stdenv.cc.targetPrefix}ld.lld -Clinker-flavor=lld-link";
+    # Necessary because otherwise we will get (useless) hardening options in front of
+    # -flavor link which will break the whole command-line processing for the ld.lld linker.
+    hardeningDisable = [ "all" ];
+
+    meta = with lib; {
+      description = "Lanzaboote UEFI stub for SecureBoot enablement on NixOS systems";
+      homepage = "https://github.com/nix-community/lanzaboote";
+      license = licenses.mit;
+      platforms = [ "x86_64-windows" ];
+    };
+  } // lib.optionalAttrs enableLint {
```

Comment:
Is it intentional that these extra attributes bypass `buildRustPackage`?  (You need to wrap the attrset merge in parens if you don't want it to — `({ … } // { … })`.